packages/manager/modules/pci/src/projects/project/kubernetes/details/service/update/translations/Messages_fr_FR.json

```diff
@@ -2,7 +2,10 @@
   "kube_service_update_common_confirm": "Confirmer",
   "kube_service_update_common_cancel": "Annuler",
   "kube_service_update_title": "Forcer la mise à jour maintenant",
+  "kube_service__minor_version_update_title": "Montée de version mineure",
   "kube_service_update_message": "La mise à jour entrainera possiblement une indisponibilité de l'ordre d'une à deux minutes pour chacun de vos noeuds. Nous utilisons la mise à jour roulante (rolling upgrade with node drain) pour rendre l'impact sur vos services exposés minime ou transparent.",
+  "kube_service_minor_version_update_message": "Grace à cette fonctionnalité, vous pouvez montez votre cluster d'une version mineure à la suivante ( {{clusterMinorVersion}} vers {{ nextMinorVersion }}). Avant de lancer cette action nous vous invitons à sauvegarder la configuration de votre cluster et valider que  les outils avec lesquels vous travaillez sont compatibles avec cette nouvelle version mineure.<br/><br/>Veuillez également vous assurer que vous avez dimensionné votre cluster avec suffisamment de noeuds de travail pour permettre une montée de version roulante (rolling upgrade). Vous ne pourrez pas revenir à la version mineure précédente.<br/><br/>Afin de minimiser le temps d'indisponibilité de chacun de vos pods et de votre service en général, nous allons évacuer ('drain') sucessivement chacun de vos noeuds, passant au suivant dès que le premier est fonctionnel (état 'ready'). Enfin, nous effectuerons la mise à jour des différents composants admin, dont votre API server.<br/><br/>L'ensemble de ces opérations prend généralement plusieurs dizaines de minutes (variable selon la typologie de votre cluster).",
```

Comment:
- Grâce
- vous pouvez monter
i think this message could be splitted in multiple translations keys to avoid the html <br /> elements

Reply:
These changes are done :)